docs/source/scheduling-state.rst

```diff
@@ -84,17 +84,17 @@ forgotten
     dereferenced from the scheduler.
 
 .. note::
-    When the ``distributed.scheduler.worker_saturation`` config value is set to ``inf``
-    (default), there's no intermediate state between ``waiting`` / ``no-worker`` and
+    Setting ``distributed.scheduler.worker_saturation`` config value to ``1.0``
+    (default) will queue excess root tasks on the scheduler in the ``queued`` state. 
```

Comment:
Please note that the default is `1.1`, not `1.0`. We should also mention that this behavior is valid for any finite value, not just the default. For example:

```suggestion
    Setting ``distributed.scheduler.worker_saturation`` config value to ``1.1``
    (default) or any other finite value will queue excess root tasks on the scheduler in the ``queued`` state. 
```

Reply:
I've changed it like what you suggested. Please tell me if there is anything else that I should do.